src/components/HomePageContent/ClusterInfo/ClusterInfo.tsx

```diff
@@ -0,0 +1,61 @@
+import clsx from 'clsx';
+
+import talents from 'src/assets/home-page-images/kha-talents.webp';
+import club from 'src/assets/home-page-images/prof-club.webp';
+import { buttonVariant } from 'src/components/ui/Button';
+import PageHeading from 'src/components/ui/PageHeading';
+import PageText from 'src/components/ui/PageText';
```

Comment:
you can short it, just write components/ui/PageText, there is an alias for this path